packages/terra-functional-testing/lib/logger/logger.js

```diff
@@ -0,0 +1,27 @@
+class Logger {
```

Comment:
Do we intent to create a logger for each package?

Reply:
I brought this question up in scrum last week, we don't currently have a re-usable Logger that can be used between packages. But each Logger we have is only a few lines of code. If we want logging in this project, at least for now, we need to create one.  A re-usable Logger would be nice for consistency between projects that utilize logging. I think it would be a nice-to-have but not a make-or-break between projects.

This would be a good war room topic or post-scrum discussion if we have time